app/models/game_company.rb

```diff
@@ -0,0 +1,3 @@
+class GameCompany < ApplicationRecord
+  has_many :video_games, -> { order(created_at: :desc) }
```

Comment:
i think if it's already ordered here you don't have to order it inside of your app/view/game_companies/index

Reply:
@georgehwho I think I got ahead of the game or even added an unnecessary feature, but that is actually supposed to be ordering the video games that are associated with each game company by their own creation times. I thought maybe it'd come in handy later, who knows though 🤷🏽‍♂️